pkg/cmd/issue/view/view_test.go

```diff
@@ -16,11 +16,37 @@ import (
 	"github.com/cli/cli/v2/pkg/cmdutil"
 	"github.com/cli/cli/v2/pkg/httpmock"
 	"github.com/cli/cli/v2/pkg/iostreams"
+	"github.com/cli/cli/v2/pkg/jsonfieldstest"
 	"github.com/cli/cli/v2/test"
 	"github.com/google/shlex"
 	"github.com/stretchr/testify/assert"
 )
 
+func TestJSONFields(t *testing.T) {
+	jsonfieldstest.ExpectCommandToSupportJSONFields(t, NewCmdView, []string{
```

Comment:
Nothing we need to change for this PR but I always struggle with the ordering of these fields needing to be synced across files.  Not the end of the world but a smell in my opinion.

Reply:
Just to be clear the order isn't programmatically required as the assertion function compares without considering order (and this is doc-ed on the command) but from a code reading point of view it's very helpful.